core/include/tee/tee_fs.h

```diff
@@ -67,4 +68,34 @@ TEE_Result tee_rpmb_fs_raw_open(const char *fname, bool create,
 bool plat_rpmb_key_is_ready(void);
 #endif
 
+/*
+ * Returns the appropriate tee_file_operations for the specified storage ID.
+ * The value TEE_STORAGE_PRIVATE will select the REE FS if available, otherwise
+ * RPMB.
+ */
+static inline const struct tee_file_operations *tee_svc_storage_file_ops(
```

Comment:
Strange indentation. I would prefer:
``` C
static inline const struct tee_file_operations *
tee_svc_storage_file_ops(uint32_t storage_id)
```

Reply:
I'll fix. We'd better have chosen `fops` instead of `file_operations`... These kind of broken lines make things unnecessarily harder to grep with meaningful output :-/

Reply:
A friend of mine prefers this style since it actually makes it easier to grep for the implementation of a function with `grep '^...`. But that was 20 years ago. :-)

Reply:
Are you getting old, Jérôme? ;-)